frontend/src/components/Registration.tsx

```diff
@@ -8,18 +10,27 @@ const layout = {
 const tailLayout = {
     wrapperCol: { offset: 4, span: 16 },
 };
+class Registration extends React.Component{
 
-const Registration = () => {
-    const onFinish = (values: any) => {
-        console.log('Success:', values);
+    onFinish = async (values: any) => {
+        axios.post("http://localhost:3000/users", values)
+        .then(res => {
+            if(res.request.statusText === 'Created') {
+                alert("Message Sent. Please check your email for confirmation!"); 
```

Comment:
Use this instead of `alert`: https://ant.design/components/message/

Reply:
Okay.